.github/workflows/release-swift.yml

```diff
@@ -0,0 +1,148 @@
+        id: version
+        run: |
+          VERSION=$(grep -o '^version = ".*"' crates/bitwarden/Cargo.toml | grep -Eo "[0-9]+\.[0-9]+\.[0-9]+")
+          echo "version=$VERSION" >> $GITHUB_OUTPUT
+
+  github-release:
+    name: GitHub Release
+    runs-on: ubuntu-22.04
+    needs: validate
+    env:
+      _PKG_VERSION: ${{ needs.validate.outputs.version }}
+    steps:
+      - name: Download BitwardenEFI artifact
+        uses: bitwarden/gh-actions/download-artifacts@62d1bf7c3e31c458cc7236b1e69a475d235cd78f
+        with:
+          workflow: build-swift.yml
+          artifacts: BitwardenFFI-${{ env._PKG_VERSION }}.xcframework
+          path: packages
+          workflow_conclusion: success
+          branch: ${{ github.event.inputs.release_type == 'Dry Run' && 'master' || github.ref_name }}
+
+      - name: Create release
+        if: ${{ github.event.inputs.release_type != 'Dry Run' }}
+        uses: ncipollo/release-action@6c75be85e571768fa31b40abf38de58ba0397db5 # v1.13.0
+        with:
+          artifacts: "packages/BitwardenFFI-${{ env._PKG_VERSION }}.xcframework"
+          commit: ${{ github.sha }}
+          tag: BitwardenFFI-v${{ env._PKG_VERSION }}
+          name: BitwardenFFI v${{ env._PKG_VERSION }}
+          body: "<insert release notes here>"
+          token: ${{ secrets.GITHUB_TOKEN }}
+          draft: true
+
+  repo-sync:
+    name: Push changed files to SDK Swift repo
+    runs-on: ubuntu-22.04
+    needs: validate
+    env:
+      _KEY_VAULT: "bitwarden-ci"
+      _BOT_EMAIL: 106330231+bitwarden-devops-bot@users.noreply.github.com
+      _BOT_NAME: bitwarden-devops-bot
+      _PKG_VERSION: ${{ needs.validate.outputs.version }}
+    steps:
+      - name: Checkout SDK repo
+        uses: actions/checkout@3df4ab11eba7bda6032a0b82a6bb43b11571feac # v4.0.0
+        with:
+          path: sdk
+
+      - name: Checkout SDK-Swift repo
+        uses: actions/checkout@3df4ab11eba7bda6032a0b82a6bb43b11571feac # v4.0.0
+        with:
+          repository: bitwarden/sdk-swift
+          path: sdk-swift
+          ref: main
+
+      - name: Login to Azure - Prod Subscription
+        uses: Azure/login@92a5484dfaf04ca78a94597f4f19fea633851fa2 # v1.4.7
+        with:
+          creds: ${{ secrets.AZURE_CI_SERVICE_PRINCIPAL }}
+
+      - name: Retrieve secrets
+        id: retrieve-secrets
+        uses: bitwarden/gh-actions/get-keyvault-secrets@62d1bf7c3e31c458cc7236b1e69a475d235cd78f
+        with:
+          keyvault: ${{ env._KEY_VAULT }}
+          secrets: "github-pat-bitwarden-devops-bot-repo-scope"
+
+      - name: Setup Git
+        working-directory: sdk-swift
+        run: |
+          git config --local user.email "${{ env._BOT_EMAIL }}"
+          git config --local user.name "${{ env._BOT_NAME }}"
+
+      - name: Update files
+        run: |
+          cp --verbose -rf sdk/languages/swift/README.md sdk-swift/README.md
+          cp --verbose -rf sdk/languages/swift/Package.swift sdk-swift/Package.swift
```

Comment:
We need to update package.swift with the new url.

Reply:
Oh, you are right!
I was looking at this and I noticed that we need to add this entire codeblock, with different values of course.
```swift
        .binaryTarget(
            name: "BitwardenFFI",
            url: "https://bwlivefronttest.blob.core.windows.net/sdk/6b8b84f-BitwardenFFI.xcframework.zip",
            checksum: "28851ab2735ecfa357c4ee4b73282e3f7b3fbb2e9258af9ddd05f160f0d774c0"),
```
Is there a good way to do this that you know of? I am trying to think of a way without spilcing up the file with something like `sed` or something similar.

The `sed` command I used is here, it works, but doesn't maintain formatting.

```bash
sed -i '' 's|.binaryTarget(name: "BitwardenFFI", path: "BitwardenFFI.xcframework")|.binaryTarget(\
    name: "BitwardenFFI",\
    url: "https://bwlivefronttest.blob.core.windows.net/sdk/6b8b84f-BitwardenFFI.xcframework.zip",\
    checksum: "28851ab2735ecfa357c4ee4b73282e3f7b3fbb2e9258af9ddd05f160f0d774c0"),|' Package.swift
```
_values are just for testing_

Reply:
Not on top of my mind.